src/styles/_services.scss

```diff
@@ -2,7 +2,119 @@
   #hero {
     height: 100vh;
     min-height: 700px;
-    background-image: url("/assets/img/services/finsecptx-service-banner-2000.jpg");
+    background-image: url("/assets/img/services/hero-phone-boot-bg-2000.jpg");
+    background-position: center;
+    background-size: cover;
+    div {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      h1 {
+        font-size: 40px;
+        font-weight: 800;
+        line-height: 120%;
+        text-transform: uppercase;
+        max-width: 590px;
+      }
+    }
+  }
+  .services-layout {
```

Comment:
Given that it's already scoped and confined with `#services { ... }` there is no need to repeat the word services.

The final output in CSS will then be `#services .services-layout` if that makes sense. On this entire services page is only one special class you call `.layout' feel free to simplify to just that. It's nothing breaking nor important.